### DISCUSSION TITLE
getting-started/configuration-overview/

### DISCUSSION THREAD

**Discussion by giscus** [go-gost/gost#368] [General]
# getting-started/configuration-overview/

https://latest.gost.run/getting-started/configuration-overview/

**Comment by rinex20** [comment]
```
sc create gost binpath= "C:\gost.exe -L :8080" start= auto
```
会出现1053错误。

**Comment by cnwhy** [comment]
如果是报 `Set-Content : 找不到接受实际参数` 貌似加上 sc 后面补全 `.exe` 就好了
```
sc.exe create gost binpath="C:\gost.exe -L :8080" start=auto
```

**Reply by rinex20** [reply]
不是报这个问题，service已经创建成功了，只是启动的时候会提示1053错误。
网上有文章说，1053是timeout的问题，可以通过配置注册表的service超时来解决，但是我修改了timeout为60000（ms），也没解决这个问题。

**Comment by 510846** [comment]
启动命令中：
-L :8080

这个参数是什么意思呢？（Gost 2.x 中并没有看到这个参数）

**Reply by 510846** [reply]
原来是默认开启了一个 http 8080 的代理，

不过我不解的是，像这样的 “-L :8080” 开启的代理服务，有实质的效果吗？ 访问 http://localhost:8080 并不能打开，这个的作用是？

**Reply by beck-8** [reply]
这根本不是用来打开的，这是当作proxy的
